owrx/wsjt.py

```diff
@@ -197,6 +209,25 @@ def getMode(self):
         return "FST4W"
 
 
+class JT4Profile(WsjtProfile):
+    availableSubmodes = ["A", "B", "C", "D", "E", "F", "G"]
+
+    def __init__(self, submode):
+        self.submode = submode
+
+    def getInterval(self):
+        return 60
+
+    def getSubmode(self):
+        return self.submode
+
+    def decoder_commandline(self, file):
+        return ["jt9", "-4", "-b", str(self.submode), "-F", "1000" , "-d", str(self.decoding_depth()), file]
```

Comment:
Is there a penalty for setting `-F` to such a high value? I would expect it to cause increased CPU load. If so, it may be a good idea to make it configurable, too.

Reply:
I have removed it. Is this possible for you to add this capability?  This is needed capability especially on uWaves where frequencies are not stable enough.
I was trying to add it but cannot implement the web part, my knowledge in this area is too low.

Reply:
I can look into it, but I'll have to get a few other things off my desk first. I do understand the need, but I really don't know how WSJT-X would react to such a big change (after all, the default is just `-F 20`), hence the question. Do you have any experience regarding the CPU usage?

Reply:
No, not really;(

Reply:
Just a kindly reminder;)

Reply:
Yeah reminders don't help a lot with allocating time.

You could experiment regarding the CPU usage in the meantime.